src/pwmodel/helper.py

```diff
@@ -298,7 +298,7 @@ def get_line(file_object, limit=-1, sep=r'\s+', pw_filter=lambda x: True, errors
     for l in file_object:
         if limit > 0 and limit <= i:
             break
-        t = re.split(sep, l.rstrip('\n'), maxsplit=1)
+        t = l.rstrip('\n').split('\t')
```

Comment:
You can change the sep to be "\s" or "\t". Some password leaks are notorious with the formats, and that's why I used sep.

Reply:
Thanks just setting `sep` to be `'\t'` would solve the problem. Closing the pull request.